Pod/Classes/CodeAttributedString.swift

```diff
@@ -208,10 +208,10 @@ open class CodeAttributedString : NSTextStorage
     
     func setupListeners()
     {
-        highlightr.themeChanged =
-            { _ in
-                    self.highlight(NSMakeRange(0, self.stringStorage.length))
-            }
+        weak var wself: CodeAttributedString? = self
+        highlightr.themeChanged = { _ in
+            wself?.highlight(NSMakeRange(0, wself?.stringStorage.length ?? 0))
```

Comment:
Good catch, but I'll prefer this pattern instead:
```suggestion
        highlightr.themeChanged =
            { [weak self] _ in
                    guard let self = self else { return }
                    self.highlight(NSMakeRange(0, self.stringStorage.length))
```

Reply:
Well, It's more safe using `guard else`

Reply:
Well, It's safer using `guard else`

Reply:
Well, It has better code extensibility.